config/services.example.json

```diff
@@ -34,9 +34,49 @@
         "error_params": {
           "scaling": 2,
           "min_th_up": 0.1,
```

Comment:
`minimal_upper_threshold` and `minimal_lower_threshold` see below the config of ensemble plugin.